### DISCUSSION TITLE
Can't rebind ConvertNext as expected

### DISCUSSION THREAD

**Discussion by diminishedprime** [google/mozc#526] [Q&A] [answered]
I'm currently trying to use mozc with fcitx. I'd like to be able to use `ctrl-n` and `crtl-p` to navigate through the available options. 

I was able to partially get this to work by setting the shortcuts for the `Prediction` mode with the command `ConvertNext`. However, these shortcuts don't work until after I press `Down` on the keyboard. 

![2021-11-26_15-42-56](https://user-images.githubusercontent.com/7953836/143661328-a3e5a20b-5b09-4c9f-9999-0f301c9289b4.png)

I tried mapping `ctrl-n` to every mode as `ConvertNext`, but this also didn't work until after I pressed `Down` on the keyboard.

![2021-11-26_15-38-00](https://user-images.githubusercontent.com/7953836/143661389-55704ada-5669-4d46-8287-5337badba848.png)

I imagine I'm missing something simple here, but I'm not sure what that is. 

Thanks

**Comment by hiroyuki-komatsu** [comment, marked as answer]
Hi, you might want to check the existing settings for {`Composition` and `Down`}, then add the same settings for {`Composition` and `ctrl-p`}.

Thanks,
